exercises/anagram/.meta/gen.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"text/template"
+
+	"github.com/exercism/xgo/gen"
+)
+
+func main() {
+	t, err := template.New("").Parse(tmpl)
+	if err != nil {
+		panic(err)
+	}
+	if err = gen.Gen("anagram", &js{}, t); err != nil {
+		panic(err)
+	}
+}
+
+type js struct {
+	Cases []struct {
+		Description string
+		Property    string
+		Subject     string
+		Candidates  []string
+		Expected    []string
+	}
+}
+
+const tmpl = `package anagram
+
+{{.Header}}
+
+type testcase struct{
+  subject string
+  candidates []string
+  expected []string
+  description string
+}
+
+var testCases = []testcase{
+  {{range .J.Cases}}
+	{
+	  subject: {{printf "%q" .Subject}},
```

Comment:
I would take a look at the whitespace at these four lines, looks like there are some tabs and spaces mixed. It does not matter in the end since the end result gets formatted anyway, but helps to use just one (I don't care which) in case tab widths differ.